test/streams.js

```diff
@@ -119,7 +119,9 @@ function test (batch = 1) {
         t.fail('should never end')
       })
 
-      feed.append(['c', 'd', 'e'])
+      setImmediate(function () {
+        feed.append(['c', 'd', 'e'])
+      })
```

Comment:
why is this needed?

Reply:
Same reason as in https://github.com/hypercore-protocol/hypercore/pull/276#discussion_r514252973